benchmarks/bench_sort.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2011 Fuji, Goro (gfx) <gfuji@cpan.org>.
+ * Copyright (c) 2019 Morwenn.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+#include <cstdlib>
+#include <ctime>
+#include <algorithm>
+#include <iostream>
+#include <vector>
+#include <gfx/timsort.hpp>
+#include "benchmarker.hpp"
+
+template <typename value_t>
+struct Bench {
+    void operator()(const std::vector<value_t> &a) const {
+        {
+            std::vector<value_t> b(a);
+
+            std::clock_t start = std::clock();
```

Comment:
I guess that this is from the original benchmarks, like several of my remarks above - looks like I never gave a proper look to the old benchmarks haha -, but IIRC `std::clock()` is rather bad for benchmarks because of implementation divergence. What you used in the other benchmark seems better overall.

https://stackoverflow.com/a/28396359/1364752

Reply:
I haven't changed the implementation of the original benchmark, just made parts of it reusable. I think that if it is to be changed, this should be done in a separate pull request.

Reply:
Just to be clear, I do not plan to improve/refactor the old benchmark. It seems to work well enough in practice, at least on GNU/Linux, so I don't think this is particularly important.